src/pages/Apps/AppCardAlby.tsx

```diff
@@ -0,0 +1,101 @@
+import { PlusIcon } from "@heroicons/react/24/outline";
+import { FC } from "react";
+import { useTranslation } from "react-i18next";
+import { App } from "../..//models/app.model";
+import AppIcon from "../../components/AppIcon";
+import { toast } from "react-toastify";
+import { instance } from "../../utils/interceptor";
+
+export type Props = {
+  appInfo: App;
+};
+
+export const AppCardAlby: FC<Props> = ({ appInfo }) => {
+  const { id, name } = appInfo;
+  const { t } = useTranslation();
+
+  const addAlbyAccountHandler = async () => {
+    const resp = await instance.get("/system/connection-info");
+
+    if (
+      resp.status !== 200 ||
+      !resp.data.lnd_admin_macaroon ||
+      !resp.data.lnd_rest_onion
+    ) {
+      toast.error(t(`appInfo.${id}.action.connection_info_error`));
+      return;
+    }
+
+    const { lnd_admin_macaroon, lnd_rest_onion } = resp.data;
+
+    const albyProvider = window.alby;
+
+    if (!albyProvider) {
+      toast.error(t(`appInfo.${id}.action.connection.hint`));
+      return;
+    }
+
+    try {
+      await albyProvider.enable();
+
+      const result = await albyProvider.addAccount({
+        name: "⚡️ Raspiblitz",
+        connector: "lnd",
+        config: {
+          adminkey: lnd_admin_macaroon,
+          url: lnd_rest_onion,
+        },
+      });
+
+      if (result.success) {
+        toast.success(t(`appInfo.${id}.action.connection.success`));
+      } else {
+        toast.error(t(`appInfo.${id}.action.connection.error`));
+      }
+    } catch (e) {
+      toast.error(t(`appInfo.${id}.action.connection.error`));
+    }
+  };
+
+  return (
+    <div className="bd-card transition-colors dark:bg-gray-800">
+      <div className="relative mt-2 flex h-4/6 w-full flex-row items-center">
+        {/* Icon */}
+        <div className="mt-4 flex w-1/4 items-center justify-center p-2">
+          <AppIcon appId={id} className="max-h-12" />
+        </div>
+        {/* Content */}
+        <div className="mt-4 flex w-3/4 flex-col items-start justify-center text-xl">
+          <h4>{name}</h4>
+          <p className="overflow-ellipsis text-base text-gray-500 dark:text-gray-200">
+            {t(`appInfo.${id}.shortDescription`)}
+          </p>
+        </div>
+      </div>
+      <div className="flex flex-row gap-2 py-4">
```

Comment:
- Please center both buttons
- Use a different Icon for Install (+ like the others) & Connect(maybe the chain or sth else? => heroicons.com/ )

Reply:
<img width="497" alt="image" src="https://github.com/cstenglein/raspiblitz-web/assets/1016218/7aeee75e-4cef-4d1d-b12f-31f130c4a6cd">
Changed the icon and centered the button. But a centered single button looks weird compared to the other boxes I think.

Reply:
<img width="494" alt="image" src="https://github.com/cstenglein/raspiblitz-web/assets/1016218/18615518-f5ba-4d0f-9551-cb5b4125037a">
You meant like this? I like the link-icon. The center alignment looks weird compared to the others boxes.

Reply:
> But a centered single button looks weird compared to the other boxes I think.

I would be happy either way, but if you think it looks weird you can revert it and it's fine for me as well :)